### PR TITLE
Resolves issue #90

### PR DESCRIPTION
**Pull request by vegitron**

### CODE REVIEW

File: tests/django16/model_regress/models.py [hunk 1]
```diff
@@ -82,3 +82,8 @@ class Model2(models.Model):
 
 class Model3(models.Model):
     model2 = models.ForeignKey(Model2, unique=True, to_field='model1')
+
+
+class Slicer(models.Model):
+    num = models.IntegerField()
+    f2 = models.IntegerField()
```
Comment (dlo) [inline tests/django16/model_regress/models.py]: Could you rename `f2` to a more descriptive name? Thanks.

File: django_pyodbc/compiler.py [hunk 2]
```diff
@@ -345,14 +345,29 @@ def _alias_columns(self, sql):
...
+        # the placeholder_data is used to prevent the variable "i" from
+        # being interpreted as a local variable in the replacement function
+        placeholder_data = { "i": i }
+        def _alias_placeholders(val):
+            i = placeholder_data["i"]
+            i += 1
+            placeholder_data["i"] = i
+            key = "_placeholder_{0}".format(i)
+            parens[key] = "%s"
+            return "%("+key+")s"
```
Comment (dlo) [inline django_pyodbc/compiler.py]: :arrow_right: `"%(" + key + ")s"`

**Comment by vegitron** [comment]
Ok, fields renamed and formatting fixed up.

**Comment by dlo** [comment]
@vegitron thanks!

**Pull request merged**